test/table.ts

```diff
@@ -2014,54 +2045,104 @@ describe('BigQuery/Table', () => {
       }),
     };
 
+    // HACK @types/sinon is missing the timer async methods
```

Comment:
So it's ok to have this here for now.  Please:
- submit an upstream PR to Definitely Typed with the fix
- reference the PR in the comment here
- file a process bug to follow up after the fix lands

We try to help maintain the community types here :)

Reply:
i filed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43148

and upon submitting the PR realized https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43139 was also open, less than 12h difference :(

Reply:
Looks like it's going to be a while until the DT PR is merged, due to an export bug in a dependency (`@sinonjs/fake-timers` proper). I've opened #655 to track it.

Reply:
`@types/sinon@9.0.0` landed, merged into this PR, and I was able to remove the shim.